app/controllers/gids_controller.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-require 'gi/client'
+require 'gids_redis/Gids'
 
-class GIController < ApplicationController
+class GidsController < ApplicationController
```

Comment:
Can we upcase `Gids` -> `GIDS` wherever possible (like `GI` is upcase in the 'before' of this diff).  You'll need to add `GIDS` to `config/initializers/inflections.rb`

Reply:
changed